kfs-purap/src/main/java/edu/arizona/kfs/module/purap/PurapWorkflowConstants.java

```diff
@@ -0,0 +1,103 @@
+package edu.arizona.kfs.module.purap;
```

Comment:
None of the KFS 3.0 code in this constants file belongs here. If you look at the KFS 3.0 and KFS 6.0 versions of this file, you will see they are quite different. The code that handles route and document status values for the Requisition document is in org.kuali.kfs.module.purap.PurapConstants (lines 128-215). I am guessing this KFS 3.0 code is not even needed.